packages/js-drive/lib/abci/handlers/beginBlockHandlerFactory.js

```diff
@@ -126,7 +126,26 @@ function beginBlockHandlerFactory(
     );
 
     if (isSimplifiedMasternodeListUpdated) {
-      await synchronizeMasternodeIdentities(coreChainLockedHeight);
+      const synchronizeMasternodeIdentitiesResult = await synchronizeMasternodeIdentities(
+        coreChainLockedHeight,
+      );
+
+      const {
+        created, updated, removed, fromHeight, toHeight,
+      } = synchronizeMasternodeIdentitiesResult;
+
+      consensusLogger.info(
+        `For SML from height ${fromHeight} to ${toHeight} ${coreChainLockedHeight}, ${created.length} masternode entries created, ${updated.length} updated, ${removed.length} removed`,
```

Comment:
`Masternode identities are synced for heights from ${fromHeight} to ${toHeight}: ${created.length} created, ${updated.length} updated, ${removed.length} removed`